pkg/services/internal/whatsonchain/service.go

```diff
@@ -56,15 +62,15 @@ func (woc *WhatsOnChain) prepareRequest(req *resty.Request) *resty.Request {
 	return req
 }
 
```

Comment:
ohhh, because you're returning a value not pointer, there is no point to hold it as pointer, you can simply use value field, which will initialize to zero value by default (so the comment on line 49 will be redundant)